### ISSUE TITLE
manual-image-processing when same overlay over side by side

### ISSUE THREAD

**Issue by yuvalbl** [AnSyn/ansyn#875]
The current design is to save current manual-process-parameters per overlay (and not per imagery / map)
This require a clear design for cases when same overlay is displayed on multiple imageries

1. I assume each imagery should be processed separately, but it should be clear to the user which is the one to be saved.
2. If 1 is correct, we should discussed a mechanism to save these manual-process-parameters per imagery, and make sure it's only a temporary state

**Comment by yuvalbl** [comment]
After discussion with @idanbarak and @asafMasa:
current behaviour is OK but the following is a bug:

1. change layout to display 2 maps
2. open same overlay on both maps
3. open tools bar and set it pinned (so it is always displayed)
4. click on manual image processing button
5. change a parameter (like: brightness)
6. select the non-active map, so it becames active

Actual behavior:
manual processing controls stays the same

Expected behavior:
manual processing controls should reflect the settings of the overlay on the active map (should change when active map is changed)

**Comment by yuvalbl** [comment]
@idanbarak @carmelorr 
Please specify the required behavior for the state I've described
![image](https://user-images.githubusercontent.com/11487010/36385082-33e3ccb4-159a-11e8-9f96-c58e7058dbb0.png)

**Comment by carmelorr** [comment]
@yuvalbl you are right. this is a bug (not only for the same overlay). please see #1223
